index.html

```diff
@@ -2,146 +2,196 @@
-                <li>html</li>
-                <li>bootstrap</li>
-                <li class="ruby">Ruby</li>
-              </ul>
-              <button class="button" type="button">See Project</button>
-            </li>
+        <h2 class="post">Multi-Post Stories</h2>
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="lang">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <button class="button" type="button">See Project</button>
+      </li>
       <li>
         <div>
-
-          <img class="image1" src="./placeholder.png" alt="image placeholder">
+          <img class="image1" src="./placeholder.png" alt="image placeholder" />
         </div>
-          <h2 class="post">Multi-Post Stories</h2>
-              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
-                required. has been the industry's standard dummy
-                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-              <ul class="lang">
-                <li>css</li>
-                <li>html</li>
-                <li>bootstrap</li>
-                <li class="ruby">Ruby</li>
-              </ul>
-              <button class="button" type="button">See Project</button>
-            </li>
+        <h2 class="post">Multi-Post Stories</h2>
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="lang">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <button class="button" type="button">See Project</button>
+      </li>
     </ul>
+  </section>
+  <section class="footer">
+    <div class=footer-text>
+      <h2 class="contact">CONTACT US</h2>
+      <p class="contact-text">&nbsp;&nbsp;&nbsp;&nbsp;If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+    </div>
+      <form action="https://formspree.io/f/xeqnejgd" method="post" class="conct-form">
+        <input type="text" name="name" id="username" placeholder="Text Input" maxlength="30" required /> <br>
+        <input type="email" name="email" id="useremail" placeholder="jeremy@gmail.com" required /><br>
+        <textarea name="message" id="usermessage" placeholder="Enter text here" maxlength="500" cols="30" rows="10"></textarea>
```

Comment:
- [ ] Good job implementing the inputs!! Now kindly add the `required` attribute to this `textarea` element so the users' submitted forms (and the information you received from them) are consistent.  💪

Reply:
- [x] This well addressed and resolved ✔️